pyodide_build/pywasmcross.py

```diff
@@ -332,6 +355,10 @@ def handle_command(line, args, dryrun=False):
         # See https://github.com/emscripten-core/emscripten/issues/8650
         if arg in ["-lfreetype", "-lz", "-lpng", "-lgfortran"]:
             continue
+        # don't use -shared, SIDE_MODULE is already used
+        # and -shared breaks it
+        if arg in ["-shared"]:
+            continue
```

Comment:
Adding a test for it also wouldn't hurt.

Reply:
This is already tested for (and the tests currently fail because they haven't been updated)